cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -1,164 +1,92 @@
 # -*- coding: utf-8 -*-
 import fauxfactory
 import pytest
+
+from cfme.cloud.availability_zone import AvailabilityZone
 from cfme.cloud.instance import Instance
-from cfme.cloud.provider import CloudProvider
-from cfme.web_ui import InfoBlock, toolbar, jstimelines
-from cfme.exceptions import ToolbarOptionGreyedOrUnavailable
-from utils import testgen
-from utils import version
+from cfme.cloud.provider.azure import AzureProvider
+from utils import testgen, version
 from utils.appliance.implementations.ui import navigate_to
-from utils.blockers import BZ
 from utils.log import logger
 from utils.wait import wait_for
 
 
-pytest_generate_tests = testgen.generate([CloudProvider], scope="module")
-
-pytestmark = [pytest.mark.tier(2)]
+pytestmark = [pytest.mark.tier(2),
+              pytest.mark.uncollectif(lambda: version.current_version() < '5.7'),
+              pytest.mark.usefixtures("setup_provider_modscope")]
+pytest_generate_tests = testgen.generate([AzureProvider], scope="module")
 
 
 @pytest.fixture(scope="module")
-def delete_fx_provider_event(db, provider):
-    logger.info("Deleting timeline events for provider name %s", provider.name)
-    ems = db['ext_management_systems']
-    ems_events_table_name = version.pick({version.LOWEST: 'ems_events', '5.5': 'event_streams'})
-    ems_events = db[ems_events_table_name]
-    with db.transaction:
-        providers = (
-            db.session.query(ems_events.id)
-            .join(ems, ems_events.ems_id == ems.id)
-            .filter(ems.name == provider.name)
-        )
-        db.session.query(ems_events).filter(ems_events.id.in_(providers.subquery())).delete(False)
+def test_instance(request, cloud_provider):
```

Comment:
**REQUIRED:** I believe you should be using setup_provider here and not cloud_provider